kview/src/main/java/dev/ikm/komet/kview/mvvm/view/genediting/PropertiesController.java

```diff
@@ -105,25 +107,23 @@ private void setupShowingPanelHandlers() {
 
         editFieldsJfxNode = FXMLMvvmLoader.make(config);
 
-        Config closePropertiesConfig = new Config(this.getClass().getResource("close-properties.fxml"))
-                .addNamedViewModel(new NamedVm("genEditingViewModel", genEditingViewModel));
-        JFXNode<Pane, ClosePropertiesController> closePropsJfxNode = FXMLMvvmLoader.make(closePropertiesConfig);
+        ConfirmationPaneCommonViewModel confirmationViewModel = new ConfirmationPaneCommonViewModel();
```

Comment:
Can you remove?
Since you want the framework to supply a view model you don't need to instantiate it here.
Once removed you may want an instance variable to reference the viewModel inside the controller.

```java
confirmationViewModel = jfxNode.getViewModel("viewModel").get();
```

Could you name it `ConfirmationViewModel` assuming you create a custom view model? 
Because this scenario is simply a one-to-one (one view and one view model) it is okay to call it ConfirmationPaneViewModel. But it's always good to make names more abstract in case there are complex scenarios such as (many views- to- one view model)